### PR TITLE
[release-1.3] fix gitlab-org wrapper

### PR DESCRIPTION
**Pull request by kadel**
## Description

fixes missing embedded dependency in gitlab-org plugin

## Which issue(s) does this PR fix

- Fixes https://issues.redhat.com/browse/RHIDP-5319

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

File: dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-gitlab-org-dynamic/package.json [hunk 1]
```diff
@@ -31,9 +31,10 @@
     "test": "backstage-cli package test --passWithNoTests --coverage",
     "clean": "backstage-cli package clean",
     "clean-dynamic-sources": "yarn clean && rm -Rf node_modules",
-    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage/plugin-catalog-backend-module-gitlab-org --override-interop default --no-embed-as-dependencies"
+    "export-dynamic": "janus-cli package export-dynamic-plugin --embed-package @backstage/plugin-catalog-backend-module-gitlab-org --embed-package @backstage/plugin-catalog-backend-module-gitlab --override-interop default --no-embed-as-dependencies"
```
[the remark this answers is not in the document]
Comment (kadel) [reply dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-gitlab-org-dynamic/package.json:34]: There are some strange things that are going on in this. When I tried to remove `--no-embed-as-dependencies` I get `Error: Unable to validate plugin entry points: TypeError: util_endpoints_2.EndpointCache is not a constructor`
I was unable to figure out why. But we are backporting fixes here, so I am not convinced that we should change this here. In 1.4 it already uses the new behavior.

[the remark this answers is not in the document]
Comment (gashcrumb) [reply dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-gitlab-org-dynamic/package.json:34]: no, I agree we shouldn't remove that flag from here.  I was just kinda surprised that the old behavior would hit this module not found error in such a similar way.

[the remark this answers is not in the document]
Comment (gashcrumb) [reply dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-gitlab-org-dynamic/package.json:34]: I guess though it'd make sense, it's the same problem, the code has a dependency on this other package and it's not available as it's been moved to peer dependencies, `embed-as-code` does some similar things here.  By adding it as embedded then the peer dependency will be erased as the CLI will embed the package code into the dynamic plugin.

**Review by gashcrumb — APPROVED** [review]
:+1:

**Comment by kadel** [comment]
not sure what is going on with tests 
```
/bin/bash: line 2: /tmp/droute-linux-amd64: Text file busy
command terminated with exit code 126
``` 
/retest

**Comment by kadel** [comment]
https://github.com/janus-idp/backstage-showcase/pull/2125 should resolve `droute` issue

**Pull request merged**